tests/Nox.Types.Tests/EntityFrameworkTests/Configuration/CountryDbConfiguration.cs

```diff
@@ -41,6 +41,7 @@ public void Configure(EntityTypeBuilder<Country> builder)
         builder.Property(e => e.PageHtml).HasConversion<HtmlConverter>();
         builder.Property(e => e.CitiesCounties).HasConversion<YamlConverter>();
         builder.Property(e => e.PhoneNumber).HasConversion<PhoneNumberConverter>();
+        builder.Property(e => e.AutoId).HasConversion<AutoNumberConverter>();
 
```

Comment:
needs also to be set as AutoGenerated

Reply:
Property set as `ValueGeneratedOnAdd()`.